test/spacesuit_cors_middleware_test.exs

```diff
@@ -0,0 +1,156 @@
+defmodule SpacesuitCorsMiddlewareTest do
+  use ExUnit.Case
+  doctest Spacesuit.CorsMiddleware
+
+  setup_all do
+    req = %{
+      :headers => %{"origin" => "http://localhost"},
+      :scheme  => "http",
+      :host    => "www.example.com",
+      :port    => 80,
+      :path    => "/matched",
+      :method  => "GET"
+    }
+    {:ok, req: req}
+  end
+
+  describe "handling non-matching request paths" do
```

Comment:
On most of these test cases I can't quite tell what the intent is. It says what it's checking but not why.